cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case2a.prm

```diff
@@ -38,20 +39,20 @@ subsection Material model
 
   subsection Multicomponent
     # TODO the benchmark uses different reference temperatures for each composition
-    #set Reference temperature         = 1573.15,473.15,473.15,473.15,473.15,1573.15,1573.15,1573.15
+    #set Reference temperature         = 1573.15,473.15,473.15,473.15,473.15,1573.15,1573.15,473.15
```

Comment:
Maybe we can delete this line and the line above, because for Case 2a the thermal expansivity is zero so the temperature does not play a role in the density anyway.